src/components/tools/tool-tile.tsx

```diff
@@ -335,16 +335,25 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
     }
   }
 
-  private handleCopyImportJson = () => {
+  private handleCopyImportJsonToClipboard = () => {
     const { appConfig, unit } = this.stores;
     const unitBasePath = appConfig.getUnitBasePath(unit.code);
     const transformImageUrl = (url: string, filename?: string) => {
       return transformCurriculumImageUrl(url, unitBasePath, filename);
     };
     const toolApiInterface = this.context;
     const toolApi = toolApiInterface?.getToolApi(this.modelId);
-    const importJson = toolApi?.exportContentAsTileJson?.({ transformImageUrl });
-    importJson && navigator.clipboard.writeText(importJson);
+    let tileJsonString = toolApi?.exportContentAsTileJson?.({ transformImageUrl });
+    if (tileJsonString) {
+      // Put all exported content in a top-level object, under key: "content",
+      // but _preserve_ existing formatting (which collapses some elements
+      // into a single line; no: indent). But DO indent w.r.t. the new key.
+      tileJsonString = (tileJsonString.slice(-1) === "\n")
+        ? tileJsonString.slice(0, -1) // Remove trailing new line char.
+        : tileJsonString;
+      tileJsonString = `{\n  "content": ${tileJsonString.replaceAll("\n", "\n  ")}` + "\n}\n";
```

Comment:
🤔 Interesting... I was imagining something that used `split()` and `join()` but this seems fine as well. I would suggest using `replace()` instead of `replaceAll()` given its broader availability in browsers and also doing something about the eslint warning. Something like:
```suggestion
      tileJsonString = `{\n  "content": ${tileJsonString.replace(/\n/g, "\n  ")}\n}\n`;
```
should work, but I haven't tried this _in situ_ so you may have to fiddle with it a bit.